### DISCUSSION TITLE
Problem with publication service in preview env ?

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#441] [Q&A] [answered]
Good Morning Lab !
Is there any problem about notice publication ? 
I get this error : 
 Error: "<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1></body></html>"
Before sending to publication service, I sent the notice to validation service and the notice got no validation error.
Bye 
Renato

**Comment by rouschr** [comment, marked as answer]
This is related to eNotices2, please see: https://enotices2.ted.europa.eu/help
